### ISSUE TITLE
in a java application when pressing alt nvda does not announce the name of the menu in focus

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14460]
### Steps to reproduce:
unfortunately this application is built internally at the organization where I work, I am also not a programmer to send code demos, the hope is that someone can replicate it in another java application.
In this application when walking with tabe there are several controls among them several buttons.
If you tab walk leave a control like a button on focus and press alt.

### Actual behavior:
nvda announces again the name of the tabed control, nvda does not announce the name of the menu that has focus.

### Expected behavior:
nvda should announce the name of the menu that has received focus.
When walking with arrows I see that the menu was opened even without nvda having announced it.
Looking at the log it looks like pressing the alt key doesn't show up in the logs.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-27325,304ea7a3

#### Windows version:
10 19041.1806
#### Name and version of other software in use when reproducing the issue:
aplication java java.exe 8.0.2010.9
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[menujava.txt](https://github.com/nvaccess/nvda/files/10261569/menujava.txt)

**Comment by mwhapples** [comment]
To add some additional context to Java Swing applications and menus. In a Java Swing application pressing alt will not normally activate the main menubar, instead you need to press f10. When a context menu is shown (eg. when right clicking something), Java access bridge does not provide suitable events for NVDA to detect a menu is opening, although if you start to cursor suitable events are sent for the various menu items getting focus and NVDA will speak correctly. So either the mentioned application has custom functionality or is using another UI toolkit such as JavaFX. Without an available example, this would be difficult to track down.
